server/webhook_parser.go

```diff
@@ -310,7 +309,11 @@ func appendCommentNotifications(wh *webhook, verb string) {
 }
 
 func quoteIssueComment(comment string) string {
-	return fmt.Sprintf("> %s", comment)
+	lines := strings.Split(comment, "\n");
```

Comment:
One last request from me actually. It's convention to not use semicolons in Go unless it's absolutely necessary (like traditional for loops syntax for example).
```suggestion
	lines := strings.Split(comment, "\n")
```